lib/src/main/java/io/ably/lib/rest/Auth.java

```diff
@@ -964,9 +964,9 @@ private TokenDetails assertValidToken(TokenParams params, AuthOptions options, b
 		return tokenDetails;
 	}
 
-	private static boolean tokenValid(TokenDetails tokenDetails) {
+	private boolean tokenValid(TokenDetails tokenDetails) {
```

Comment:
@paddybyers Why make these methods instance-wide? As far as I can see they don't use instance-specific stuff in their implementations :D

Reply:
`timeDelta` is changed to be instance-wide in this commit.